public/javascripts/SVValidate/src/keyboard/Keyboard.js

```diff
@@ -58,6 +58,13 @@ function Keyboard(menuUI) {
                     menuUI.agreeButton.removeClass("validate");
                     menuUI.notSureButton.removeClass("validate");
                     break;
+                // "h" key
+                case 72:
+                    svv.labelControl.hideLabel();
+                    svv.tracker.push("KeyboardShortcut_HideLabel", {
```

Comment:
If we are unhiding the label, we should be recording "KeyboardShortcut_UnhideLabel"

Reply:
Fixed